### PR TITLE
Add swipe component and gesture recognizers

### PR DESCRIPTION
**Pull request by byme8**
This pull request adds the `` Swipe `` control that works with mobile scrolling as expected. Basically, you can add it and have the ability to swipe left and right to show items under it. Also, it has two new gesture recognizers, `` PanGestureRecognizer `` and `` TapGestureRecognizer ``, which are utilized to make the `` Swipe `` control work.

You can have a look at the demo in the attached video:
https://user-images.githubusercontent.com/5927686/226204800-5de3ca40-076d-4624-8f7b-54c0f7e5ab5f.mp4

### CODE REVIEW

File: src/Avalonia.Labs.Controls/SwipeView/Swipe.cs [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    private readonly ContentPresenter _rightContainer;
+    private readonly ContentPresenter _leftContainer;
+    private readonly ContentPresenter _bodyContainer;
+    private readonly TransformOperationsTransition _transition;
+
+    private double _initialX;
+    private double _currentX;
+    private CancellationTokenSource? _tokenSource;
+    private ICommand? _command;
+    private object? _commandParameter;
```
Comment (emmauss) [inline src/Avalonia.Labs.Controls/SwipeView/Swipe.cs:74]: _command and _commandParameter aren't used

File: src/Avalonia.Labs.Controls/SwipeView/Swipe.cs [hunk 2]
```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Threading;
+using System.Windows.Input;
```
Comment (emmauss) [inline src/Avalonia.Labs.Controls/SwipeView/Swipe.cs:3]: This using isn't used

File: src/Avalonia.Labs.Controls/SwipeView/Swipe.cs (hunk 1 above)
Comment (byme8) [reply src/Avalonia.Labs.Controls/SwipeView/Swipe.cs]: Fixed.

**Comment by emmauss** [comment]
Sorry for the late review.

**Pull request merged**